src/System.IO.IsolatedStorage/src/project.json

```diff
@@ -1,29 +1,57 @@
 {
   "frameworks": {
-    "netcore50": {
+    "netstandard1.7": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1",
+        "System.Collections": "4.0.10",
+        "System.Diagnostics.Debug": "4.0.10",
+        "System.Diagnostics.Tools": "4.0.0",
+        "System.IO": "4.0.10",
+        "System.IO.FileSystem": "4.0.0",
+        "System.IO.FileSystem.Primitives": "4.0.0",
+        "System.ObjectModel": "4.0.10",
+        "System.Linq": "4.0.0",
+        "System.Reflection": "4.1.1-beta-devapi-24422-01",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.2.0-beta-devapi-24422-01",
+        "System.Security.Cryptography.Algorithms": "4.2.1-beta-devapi-24422-01",
+        "System.Threading": "4.0.12-beta-devapi-24422-01",
+        "System.Threading.Tasks": "4.0.10"
+      },
+      "imports": [
+        "dotnet5.8"
+      ]
+    },
+    "uap10.1": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.0.1",
         "Microsoft.NETCore.Targets": "1.0.1",
-        "Microsoft.TargetingPack.Private.NETNative": "1.0.1-beta-24415-00",
+        "Microsoft.TargetingPack.Private.NETNative": "1.0.1-beta-24422-00",
```

Comment:
I suspect we don't actually need the NETNative targeting pack reference here. We should probably remove it.